### DISCUSSION TITLE
ImageMagick

### DISCUSSION THREAD

**Discussion by shockergit** [d2phap/ImageGlass#1262] [Q&A] [answered]
Is Image glass compatible with ImageMagick. If it is then is it possible to add some feature of imagemagick to imageglass like svg, and image convertion tool, draw, shatpen etc...

**Comment by shockergit** [comment]
https://imagemagick.org/index.php

**Comment by d2phap** [comment, marked as answer]
ImageGlass uses Magick.NET to read image files, Magick.NET is a C# wrapper library of ImageMagick.
But I use it mostly for viewing because ImageGlass focuses on viewing features, not editing.

Actually, editing is quite complicated, so choose the easier path.

**Reply by shockergit** [reply]
Ok then im ok with installing the portable version of imagemagick. So you are planning to add svg support from magick in version 9.

**Reply by d2phap** [reply]
IG already supports rasterize SVG (not scalable). V9 will have full support
